src/lib/task.ts

```diff
@@ -335,7 +337,7 @@ export class Task<Ctx, Renderer extends ListrRendererFactory> extends Subject<Li
 
         if (this.listr.options?.exitAfterRollback !== false) {
           // Do not exit when explicitly set to `false`
-          throw new Error(this.title)
+          throw new Error(`${this.title}`)
```

Comment:
This already a string. Note to self.

Reply:
According to what I see in the constructor, title has the potential to be `undefined` at least _immediately after instantiation_ 

Should we set it to an empty string with a class field initializer?

Reply:
I have to check this more throughly because it might break the default renderer. Maybe throw the error with ?? task with no title.